doc/mgr/cephadm.rst

```diff
@@ -101,3 +101,68 @@ You can remove a broken host from management with::
 You can disable this health warning with::
 
   ceph config set mgr mgr/cephadm/warn_on_failed_host_check false
+
+
+Troubleshooting
+---------------
+
+Sometimes, there is a need to investigate why a cephadm command failed or why
+a specific service no longer runs properly.
+
+As cephadm deploys daemons as containers, troubleshooting daemons is slightly
+different. Here are a few tools and commands to help investigating issues.
+
+Gathering log files
+^^^^^^^^^^^^^^^^^^^
+
+Please use journalctl to gather the log files of all daemons:
+
+.. note:: Note, By default, cephadm now stores logs in journald. Which means
+that you probably won't find damon logs in ``/var/log/ceph/``
+
+To read the log file of one specific daemon, run::
+
+    cephadm logs --name <name-of-daemon>
+
+To fetch all log files of all daemons on a given host, run::
+
+    for name in $(cephadm ls | jq -r '.[].name') ; do
+      cephadm logs --name "$name" > $name;
+    done
+
+Collecting systemd status
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To print the state of a systemd unit, run::
+
+      systemctl status "ceph-$(cephadm shell ceph fsid)@<service name>.service";
+
+
+To fetch all state of all daemons of a given host, run::
+
+    fsid="$(cephadm shell ceph fsid)"
+    for name in $(cephadm ls | jq -r '.[].name') ; do
+      systemctl status "ceph-$fsid@$name.service" > $name;
+    done
+
+
+List all downloaded Container images
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I'm not sure what you've got set for RST builds here, but this underline should be the same length as the title